models/public/pspnet-pytorch/pspnet-pytorch.md

```diff
@@ -0,0 +1,85 @@
+# pspnet-pytorch
```

Comment:
@jkamelin please rename model description file to README.md according the latest changes in OMZ